packages/sync-actions/src/product-actions.js

```diff
@@ -807,3 +804,15 @@ export function actionsMapMasterVariant(oldObj, newObj) {
 
   return []
 }
+
+export function actionsMapPriceMode(diff) {
+  const setPriceModeAction = function setPriceModeAction(priceMode) {
+    return {
+      action: 'setPriceMode',
+      priceMode: priceMode[0],
+    }
+  }
+  if (diff.priceMode && diff.priceMode.length)
+    return [setPriceModeAction(diff.priceMode)]
+  return []
+}
```

Comment:
this could be replaced by adding action to `baseActionsList` sth like this 
```
  { action: 'setPriceMode', key: 'priceMode', actionKey: 'priceMode' },
```